storage/innobase/row/row0import.cc

```diff
@@ -3739,12 +3742,13 @@ dberr_t row_import_for_mysql(dict_table_t *table, dd::Table *table_def,
     return (row_import_cleanup(prebuilt, trx, err));
   }
 
-  /*a For encrypted tablespace, set encryption information. */
-  if (FSP_FLAGS_GET_ENCRYPTION(fsp_flags)) {
-    err = fil_set_encryption(
-        table->space,
-        cfg.m_is_keyring_encrypted ? Encryption::KEYRING : Encryption::AES,
-        table->encryption_key, table->encryption_iv);
+  /* For encrypted tablespace, set encryption information.
+   * If it is keyring encryption encryption iv was already read in fil_ibd_open
```

Comment:
Leading * column comment style not common in InnoDB

Reply:
done